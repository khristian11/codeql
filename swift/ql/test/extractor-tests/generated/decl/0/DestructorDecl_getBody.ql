// generated by codegen/codegen.py
import codeql.swift.elements
import TestUtils

from DestructorDecl x
where toBeTested(x)
select x, x.getBody()
