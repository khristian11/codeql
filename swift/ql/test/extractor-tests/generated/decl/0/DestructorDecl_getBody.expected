| declarations.swift:11:7:11:7 | deinit | declarations.swift:11:7:11:7 | { ... } |
| declarations.swift:48:3:50:3 | deinit | declarations.swift:48:10:50:3 | { ... } |
